### DISCUSSION TITLE
How to backup an image using macOS

### DISCUSSION THREAD

**Discussion by ScreamZ** [MichaIng/DietPi#6292] [Q&A]
Hello,

I just took a barebone Debian for orangepi3 LTS. I've burned my SD card with it, and when it booted, I've ran the custom image script on the board to get a debian cleaned image of dietpi.

Then I've rebooted, I've installed docker.

Now i want to create an image that i can burn to create other clone of this setup.
I've found : dietpi-imager (bash -c "$(curl -sSfL 'https://raw.githubusercontent.com/MichaIng/DietPi/master/.build/images/dietpi-imager')")

But I'm running macOS on my main computer and I'm not sure it's the way to deal with.
I know i can just clone the full SD card, but i would get a 32 GB file whereas i only use maybe 4/5 gB?

How to proceed ?

**Comment by MichaIng** [comment]
macOS can indeed not run this script. Easiest is probably to spin up a VM and pass through the SD card/USB card reader to it. We have DietPi appliances for quite some virtualizers which run on macOS and should be easy/fast to spin up.

**Reply by ScreamZ** [reply]
I'm not sure to understand, maybe mounting a docker on my macOS could help ?

Also using dietpi-imager will the image become small ? (like real size it takes) ou will remain big as the SD card it's on ?

Thanks :)

**Reply by MichaIng** [reply]
Something like this:
- https://www.virtualbox.org/wiki/Downloads
- https://mac.getutm.app/
- https://www.vmware.com/uk/products/fusion.html
- https://www.parallels.com/

And then use the respective VM appliance from our download page: https://dietpi.com/#download

A Debian or Ubuntu Docker container would do as well (the script works on any APT-based distro), as long as it has raw access to the SD card.

> Also using dietpi-imager will the image become small ?

Yes, that is the main purpose of the script. The filesystem + partition will be minimised, and the smallest possible image created from it + 7z-compressed.
